src/components/Quote.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from 'react';
+import './css/quote.css';
+
+const Quote = () => {
+  const [data, setData] = useState([]);
+  const [hasError, setHasError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+  useEffect(() => {
+    setIsLoading(true);
+    const fetchDate = async () => {
+      try {
+        const res = await fetch('https://api.api-ninjas.com/v1/quotes?category=leadership', {
+          headers: {
+            'X-Api-Key': 'n7v9CVhv4b6yTvQajuHO4A==yWA06hqLK6YEYJID',
+          },
+        });
+        const json = await res.json();
+        setData(json[0]);
+      } catch (error) {
+        setHasError(true);
+      }
+      setIsLoading(false);
+    };
+    fetchDate();
+  }, [setData, isLoading]);
```

Comment:
- [ ] because your dependency array has `setData and isLoading` so it is making the useEffect run infinitely because isLoading state will keep changing, so you can make this an empty array `[ ]` so it runs once your app renders